public/app/features/alerting/alert_tab_ctrl.ts

```diff
@@ -79,7 +79,7 @@ export class AlertTabCtrl {
   getAlertHistory() {
     this.backendSrv.get(`/api/annotations?dashboardId=${this.panelCtrl.dashboard.id}&panelId=${this.panel.id}&limit=50`).then(res => {
       this.alertHistory = _.map(res, ah => {
-        ah.time = moment(ah.time).format('MMM D, YYYY HH:mm:ss');
+        ah.time = this.dashboardSrv.dash.formatDate(ah.time, 'MMM D, YYYY HH:mm:ss');
```

Comment:
dashboardSrv.dash is private (should be if it is not), call getCurrent to get dashboard I think